static/bidder-info/xeworks.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "http://prebid-{{.Host}}.xe.works/?pid={{.SourceId}}"
+endpoint: "http://prebid-srv.xe.works/?pid={{.SourceId}}&host={{.Host}}"
 maintainer:
   email: "team@xe.works"
```

Comment:
sent an email to this address for verification. Please expect an email from `prebid@microsoft.com` and reply with `received`. thanks!

Reply:
email was replied. thanks!